elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -518,8 +534,14 @@ export class PeerPool extends EventEmitter {
 		return peers;
 	}
 
-	public getUniqueOutboundConnectedPeers(): ReadonlyArray<P2PPeerInfo> {
-		return getUniquePeersbyIp(this.getAllConnectedPeerInfos(OutboundPeer));
+	public getConnectedPeersByIP(
```

Comment:
Instead of returning count simply return if a peer outbound exists or not because the count will always be one if we never allow the second outbound connection